doc/xds-test-descriptions.md

```diff
@@ -745,3 +745,36 @@ the forwarding rule port `80` and the URL map host rule `myservice`.
 1. No traffic goes to backends when configuring the target URI
 `xds:///myservice`, the forwarding rule port `80` and the host rule 
 `myservice::80`.
+
+### outlier_detection
+This test verifies that the client applies the outlier detection configuration
+and temporarily drops traffic to a server that fails requests.
+
+Client parameters:
+
+1.  --num_channels=1
+2.  --qps=100
+
+Load balancer configuration:
+
+1.  One MIG with five backends, with a `backendService` configuration with the
```

Comment:
Every other test case in this file uses MIGs.

Reply:
For most tests, this is not true anymore. If this interop test is implemented in the new framework, it will only be NEG. Practically both NEG and MIG serve the same purpose: providing the list of endpoints and their statuses. I wish there was a common term for NEG and MIG.

Reply:
What I mean is that I would prefer to have this description be consistent with the other descriptions in the same file. Updating the whole file to use terminology that matches the new framework could be a separate change.

Reply:
Up to you. IMO it's "consistent" vs "incorrect".